### PR TITLE
test: Fix running tests locally

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Running all tests on a local MacBook Pro fails. This is fixed now.

#skip-changelog

## :bulb: Motivation and Context

We want to be able to run all the tests locally.

## :green_heart: How did you test it?
Running the tests locally.

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the docs if needed
- [x] Review from the native team if needed
- [x] No breaking changes

### CODE REVIEW

File: Sources/Sentry/SentryCrashIntegration.m [hunk 1]
```diff
@@ -98,11 +98,18 @@ - (void)installWithOptions:(nonnull SentryOptions *)options
...
+        BOOL isInstallationNil = NO;
+        if (installation == nil) {
+            SentryFrameInAppLogic *frameInAppLogic =
+                [[SentryFrameInAppLogic alloc] initWithInAppIncludes:self.options.inAppIncludes
+                                                       inAppExcludes:self.options.inAppExcludes];
+
+            installation =
+                [[SentryCrashInstallationReporter alloc] initWithFrameInAppLogic:frameInAppLogic];
+
+            isInstallationNil = YES;
```
Comment (marandaneto) [inline Sources/Sentry/SentryCrashIntegration.m:110]: `installation` just got a new instance and it's not `nil` anymore, but the flag is called `isInstallationNil=yes`, that's odd.
maybe `wasInstallationNil` then?

Comment (marandaneto) [reply Sources/Sentry/SentryCrashIntegration.m:110]: this is a guard to call `sendAllReports`, so maybe we can even use a better name, eg, `isInstalled`, or `canSendReports`

Comment (philipphofmann) [reply Sources/Sentry/SentryCrashIntegration.m]: I agree. I renamed it thanks.

**Review by marandaneto — APPROVED** [review]
lgtm

**Pull request merged**